src/api/orgs/copilot.rs

```diff
@@ -0,0 +1,117 @@
+use super::*;
+
+#[derive(serde::Serialize)]
+pub struct CopilotHandler<'octo, 'r> {
+    #[serde(skip)]
+    handler: &'r OrgHandler<'octo>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    per_page: Option<u8>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    page: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    since: Option<chrono::DateTime<chrono::Utc>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    until: Option<chrono::DateTime<chrono::Utc>>,
+}
+
+impl<'octo, 'r> CopilotHandler<'octo, 'r> {
+    pub fn new(handler: &'r OrgHandler<'octo>) -> Self {
+        Self {
+            handler,
+            per_page: None,
+            page: None,
+            since: None,
+            until: None,
+        }
+    }
+
+    /// Results per page (max 100).
+    pub fn per_page(mut self, per_page: impl Into<u8>) -> Self {
+        self.per_page = Some(per_page.into());
+        self
+    }
+
+    /// Page number of the results to fetch.
+    pub fn page(mut self, page: impl Into<u32>) -> Self {
+        self.page = Some(page.into());
+        self
+    }
+
+    // Show usage metrics since this date.
+    // This is a timestamp in ISO 8601 format (YYYY-MM-DDTHH:MM:SSZ).
+    // Maximum value is 28 days ago.
+    pub fn since(mut self, since: chrono::DateTime<chrono::Utc>) -> Self {
+        self.since = Some(since);
+        self
+    }
+
+    // Show usage metrics until this date.
+    // This is a timestamp in ISO 8601 format (YYYY-MM-DDTHH:MM:SSZ) and should not preceed the since date if it is passed.
+    pub fn until(mut self, until: chrono::DateTime<chrono::Utc>) -> Self {
+        self.until = Some(until);
+        self
+    }
+
+    // Retrieve copilot metrics for the entire organization
+    pub async fn metrics(
+        self,
+    ) -> crate::Result<crate::Page<Vec<crate::models::orgs_copilot::metrics::CopilotMetrics>>> {
+        let route = format!("/orgs/{org}/copilot/metrics", org = self.handler.owner);
+
+        self.handler.crab.get(route, Some(&self)).await
+    }
+
+    // Retrieve copilot metrics for a specific team within the organization
+    pub async fn metrics_team<T: ToString>(
```

Comment:
The `_team` functions have a generic `<T: ToString>` so you can pass both a `&'a str` (in most cases, people only look for one organization and it's most likely `'static`) as well as a `String`. 

Most APIs in the codebase seemed to want a String but it didn't make sense to me to take something that is relatively constant, so this is more developer sugar than anything else.